### ISSUE TITLE
Add exectution time to dashboard

### ISSUE THREAD

**Issue by antoinerabany** [wiremind/remoulade#126]
It would be nice to the following info about a message:
 - if it's pending, the duration since it's enqueuing (enqueued x min ago)
 - if t's started, the duration since it's started (started x min ago)
 - if it's Error or success, the execution time,
 - if it's canceled display nothing.

I think the simplest way to do this is to store`enqued_datetime`, `started_datetime` and `end_datetime` in the state. The thing is when you are in the `after_process_message` event you do not have access to `enqued_datetime`, so we will need to update the state instead of overwrite it.

I think we should use redis Hashes to do this. 
Please read https://redis.io/topics/data-types-intro and take a look at https://redis.io/commands#hash

Basically you will HGETALL to get the state and HMSET to set the values. The thing is there will be no equivalent of MGET, so you will need to use a redis transaction (https://redis.io/topics/transactions). You can see an examples of pipelines in results redis backend (`with self.client.pipeline() as pipe`).

Also store the priority of the message in the state, I don't fell like another MR for such a small change.
